datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -250,14 +285,75 @@ protected void restartDeadTasks() {
     }
   }
 
+  /**
+   * Returns the number of tasks yet to be stopped.
+   */
+  int getToStopTaskCount() {
```

Comment:
nit: rename getTasksToStopCount() if you changed the variable name from _toStopTasks to _tasksToStop